src/features/commands/poll.js

```diff
@@ -0,0 +1,145 @@
+                db.getPolls(interaction.guild.id).then(polls => {
+                    if (!polls.length) {
+                        interaction.reply({
+                            content: 'There are no polls in this server.',
+                            ephemeral: true
+                        });
+                        return;
+                    }
+                    const title = polls[0].polls,
+                        listEmbed = new MessageEmbed()
+                            .setColor('#0099ff')
+                            .setTitle('All polls in this server')
+                            .setDescription(title.map((pollTitle) => `⦿ [${pollTitle.split(':')[0]}](https://strawpoll.com/${pollTitle.split(':')[1]})`).join('\n'));
+                    interaction.reply({
+                        embeds: [listEmbed],
+                        ephemeral: true,
+                    })
+                });
+                break;
+            case "results":
+                const pollId = interaction.options.getString("poll-id", true);
+
+                db.getPolls(interaction.guild.id).then(polls => {
+                    if (!polls.length) {
+                        interaction.reply({
+                            content: 'There are no polls in this server.',
+                            ephemeral: true
+                        });
+                        return;
+                    }
+                    strawpoll.getStrawpollResults(pollId).then(resp => {
+                        const quickChartClient = new QuickChart();
+                        let answers = [];
+                        let votes = [];
+                        let stringEmbed = '';
+
+                        for (let i = 0; i < resp.pollAnswersArray.length; i++) {
+                            answers.push(resp.pollAnswersArray[i].answer);
+                            votes.push(resp.pollAnswersArray[i].votes);
+                            stringEmbed += `Votes for choice \`${resp.pollAnswersArray[i].answer}\`: ${resp.pollAnswersArray[i].votes}\n`;
+                        }
+
+                        quickChartClient.setConfig({
+                            type: 'bar',
+                            data: { labels: answers, datasets: [{ label: 'Votes', data: votes }] },
+                        });
+
+                        quickChartClient.setBackgroundColor("#ffffff");
+
+                        const resultsEmbed = new MessageEmbed()
+                            .setColor("#0099ff")
+                            .setTitle("Poll results")
+                            .addFields(
+                                { name: 'In text', value: stringEmbed },
+                                { name: 'In a chart', value: `[Chart image link](${quickChartClient.getUrl()})` }
+                            )
+                            .setImage(quickChartClient.getUrl());
+
+                        interaction.reply({ embeds: [resultsEmbed], ephemeral: true });
```

Comment:
This message shouldn't be ephemeral as people usually want to share and show polls, thus hiding it makes this feature far less accessible.
```suggestion
                        interaction.reply({ embeds: [resultsEmbed]});
```